ZenPacks/zenoss/Import4/bin/src_imp4mariadb.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
+#
+# The script is the starter script for imp4mariadb container
+# It runs as root
+# There maybe multiple instances running at the same time
+# task(s): perfdata conversion
+#
+echo "$0 Running ..."
+
+export VOL_D='/import4'
+export PYTHONPATH=$PYTHONPATH:$VOL_D/pkg
+export PATH=$PATH:$VOL_D/pkg/bin
+
+# common block
+progdir=$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )
+. "$progdir/utils.sh"
+
+# after a cycle
+# depending on serviced to restart the service
+
+# pre install in each container
+which rrdtool || ( [[ -f /import4/pkg/bin/install_rrdtool.sh ]] && /import4/pkg/bin/install_rrdtool.sh )
```

Comment:
Yes.. because it is not included in the zope image. But, once the container is up, then it will stay there.